determined-ai/determined
# fix: return projects in GetProjectsByUserActivity when no experiments exist [WEB-767]

## Description

fix: return projects by activity when no experiments exist WEB-767

Currently, the `GetProjectsByUserActivity` endpoint will not return any projects if no experiments exist in the database. This PR contains a query update to fix the issue. 



## Test Plan

1. Start a Determined instance with no experiments in the DB 
2. Create a workspace
3. Create a project
4. Use the swagger API to create an activity entry `/api/v1/user/activity` for the project in (3)
5. Use the swagger API to call GetProjectsByUserActivity  `/api/v1/user/projects/activity` 
6. Ensure that the project created in (3) is returned. 



## Commentary (optional)





## Checklist

~- [ ] User-facing API changes need the "User-facing API Change" label.~
~- [ ] Release notes should be added as a separate file under `docs/release-notes/`.~
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
~- [ ] Licenses should be included for new code which was copied and/or modified from any external code.~

## Ticket
WEB-767






[WEB-767]: https://determinedai.atlassian.net/browse/WEB-767?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
